demos/storybook/storybook/stories/score-card.tsx

```diff
@@ -217,15 +215,14 @@ storiesOf('ScoreCard', module)
                             label={'Score'}
                             iconSize={48}
                             iconColor={Colors.green[500]}
-                            value={98}
-                            units={'/100'}
+                            ChannelValueProps={{ value: 98, units: '/100' }}
                             icon={A}
                         />,
                     ]}
                 </HeroBanner>
             }
         >
-            <View style={{ justifyContent: 'center' }}>
+            <View style={{ justifyContent: 'space-evenly', flex: 1 }}>
```

Comment:
Still curious about this as I don't think 'space-evenly' is the desired look of these ILIs (I believe they are supposed to be smushed together without gaps between them). Do you have a before and after for this?

This isn't crucial since this is only an internal tool, but would be nice to at least know what's going on here or have an issue to track it.

Reply:
I don't have before and after. I'll pull down an older commit to see what's going on. Some part of me thinks this is a storybook only issue because `justifyContent: 'center'` still works just fine in the showcase.